sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -945,6 +953,26 @@ public void setPropagationContext(final @NotNull PropagationContext propagationC
     return new Scope(this);
   }
 
+  @Override
+  public void setLastEventId(@NotNull SentryId lastEventId) {
+    this.lastEventId = lastEventId;
+  }
+
+  @Override
+  public @NotNull SentryId getLastEventId() {
```

Comment:
It's a bit weird but [JS](https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md#deprecate-hub) removed this feature altogether:
> REMOVED - Use event processors or beforeSend instead

But to be honest I still think we need it though 😅

Reply:
If we intend to remove it, we should deprecate it for a while and only remove this in a later major, not in `8.x`.